lib/omni_api/resources/user_authorization_error_handler.rb

```diff
@@ -0,0 +1,12 @@
+module OmniApi
+  module Resources
+    class UserAuthorizationErrorHandler
+      def handle(error, connection, arguments)
+        raise error unless error.is_a?(ActiveResource::UnauthorizedAccess)
+        token = Oauth2::Token.refresh_for(OmniApi.config.user_refresh_token) rescue nil
+        raise error unless token && token.persisted?
```

Comment:
This looks weird, what will the use case be? 

I prefer the simple `refresh_token!` at least we know the actual error.